espnet/nets/pytorch_backend/e2e_asr_mulenc.py

```diff
@@ -0,0 +1,612 @@
+#!/usr/bin/env python3
+
+"""Define e2e module for multi-encoder network."""
+# Copyright 2017 Johns Hopkins University (Shinji Watanabe)
+# Copyright 2017 Johns Hopkins University (Ruizhi Li)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+
+from __future__ import division
+
+import argparse
+import logging
+import math
+import os
+
+import editdistance
+
+import chainer
+import numpy as np
+import six
+import torch
+
+from itertools import groupby
+
+from chainer import reporter
+
+from espnet.nets.asr_interface import ASRInterface
+from espnet.nets.e2e_asr_common import label_smoothing_dist
+from espnet.nets.pytorch_backend.ctc import ctc_for
+from espnet.nets.pytorch_backend.nets_utils import pad_list
+from espnet.nets.pytorch_backend.nets_utils import to_device
+from espnet.nets.pytorch_backend.nets_utils import to_torch_tensor
+from espnet.nets.pytorch_backend.rnn.attentions import att_for
+from espnet.nets.pytorch_backend.rnn.decoders import decoder_for
+from espnet.nets.pytorch_backend.rnn.encoders import encoder_for
+from espnet.nets.scorers.ctc import CTCPrefixScorer
+from espnet.utils.cli_utils import strtobool
+
+CTC_LOSS_THRESHOLD = 10000
+
+
+class Reporter(chainer.Chain):
```

Comment:
If this reporter is completely the same to the one in `e2e_asr.py`, please import it.